index.js

```diff
@@ -27,16 +27,15 @@ expressApp.get('/_ah/start', (req, res) => {
 expressApp.use(bodyParser.json(), app);
 
 // Start server
-expressApp.listen(process.env.PORT || 8080);
-
+expressApp.listen(process.env.PORT || 8080, () => console.log('Listening at port 8080'));
```

Comment:
That log is incorrect. Its listening on the port provided by the env var by default.

Reply:
Reverted change.